src/systems/userinput/resolve-action-sets.js

```diff
@@ -38,7 +38,10 @@ export function resolveActionSets() {
   userinput.toggleSet(sets.leftHandHoldingCamera, leftHand.held && leftHand.held.components["camera-tool"]);
   userinput.toggleSet(sets.leftHandHoldingInteractable, leftHand.held);
 
-  userinput.toggleSet(sets.rightHandHoveringOnNothing, !rightRemote.held && !rightRemote.hovered && !rightHand.held && !rightHand.hovered);
+  userinput.toggleSet(
```

Comment:
Did you actually mean to make this change in this PR, thought this would already be accounted for by gregs menu fix PR

Reply:
This was to fix a linting issue but Greg just fixed this on master.